src/UdApi.ts

```diff
@@ -179,15 +182,19 @@ export default class Udapi extends NamingService {
   }
 
   async registryAddress(domain: string): Promise<string> {
-    const serviceName = findNamingServiceName(domain);
+    const tld = domain.split('.').pop();
+    if (tld === undefined) {
+      throw new ResolutionError(ResolutionErrorCode.UnsupportedService, {
+        namingService: tld,
+      });
+    }
 
-    if (serviceName === 'CNS') {
-      return NetworkConfig.networks[1].contracts.Registry.address;
-    } else if (serviceName === 'ZNS') {
-      return Udapi.ZnsRegistryMap[1];
+    if (tld === 'crypto') {
```

Comment:
In future .crypto domain may be stored in new registry contract. I guess it's better to throw an error for now as it was before the change. We will figure out the best way later, possible we will remove UDAPI provider at all.